articles/automatic-software-install-in-fleet.md

```diff
@@ -73,7 +73,7 @@ SELECT 1 FROM apps WHERE name = '<SOFTWARE_TITLE_NAME>' AND version_compare(bund
 SELECT 1 FROM programs WHERE name = '<SOFTWARE_TITLE_NAME>' AND version_compare(version, '<VERSION>') >= 0;
 ```
 
-### Ubuntu (deb)
+### Debian-based (deb)
```

Comment:
Maybe keep the heading as-is and add text indicating that this works on other Debian-based (Debian itself, Mint, etc.) distros, and same thing with RPM? Also probably want to change L62 to be more generic.

Reply:
I added one more line to enumerate some of the supported OSs (didn't change back to Ubuntu because the query is actually more broad).

Reply:
And the new automatic install feature is using the "Debian-based" and "RPM-based" terms.

Reply:
Fair enough. Maybe we drop the (deb) then?

Non-blocking feedback though.

Reply:
I think it makes sense to keep `deb` and `rpm` extensions as we use them in UI.